src/pages/microblocks/[id].vue

```diff
@@ -5,18 +5,30 @@
 
   <page-header>
     Microblock
+
+    <template v-if="!microblockFound">
+      not found
+    </template>
+
     <template #tooltip>
       {{ microblocksHints.microblock }}
     </template>
   </page-header>
-  <template v-if="!isLoading">
+  <template v-if="!isLoading && microblockFound">
     <microblock-details-panel
       v-if="microblockDetails"
       :microblock-details="microblockDetails"/>
 
     <microblock-transactions-panel/>
   </template>
-  <loader-panel v-else/>
+  <loader-panel v-else-if="isLoading"/>
```

Comment:
I personally I prefer to have more flat logic to ease of readability - means not using else-if if possible. But I am fine with this solution if you see things differently

Reply:
Well I could make one more nesting level but I'm not sure if it's actually better for the code readability. @michele-franchi wdyt?

Reply:
What would be the other option? To simply use v-if instead of v-else-if?
In any case let's add what we pick to the code guidelines if it isn't already there.

Reply:
No need to panic and rewrite all guidelines :D I started the sentence with my personal preference. However, I will come up with an example.